server/controllers/families/deleteFamilyById.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response, NextFunction } from 'express';
+import { Family } from '../../database/models';
+
+const deleteFamilyById = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const { id } = req.params;
+  try {
+    const deletedFamily = await Family.destroy({
+      where: {
+        id,
+      },
+    });
+    if (!deletedFamily) {
+      res.status(400).json({
+        message: "The family you are trying to delete doesn't exist",
+      });
```

Comment:
Why do you handle the error response here? throw the error so the server error will handle it.